plugins/chat/assets/javascripts/discourse/components/chat-live-pane.hbs

```diff
@@ -1,148 +1,115 @@
-        @message={{message}}
-        @canInteractWithChat={{this.canInteractWithChat}}
-        @details={{this.details}}
-        @chatChannel={{this.chatChannel}}
-        @setReplyTo={{action "setReplyTo"}}
-        @replyMessageClicked={{action "replyMessageClicked"}}
-        @editButtonClicked={{action "editButtonClicked"}}
-        @selectingMessages={{this.selectingMessages}}
-        @onStartSelectingMessages={{this.onStartSelectingMessages}}
-        @onSelectMessage={{this.onSelectMessage}}
-        @bulkSelectMessages={{this.bulkSelectMessages}}
-        @fullPage={{this.fullPage}}
-        @afterReactionAdded={{action "reStickScrollIfNeeded"}}
-        @isHovered={{eq message.id this.hoveredMessageId}}
-        @onHoverMessage={{this.onHoverMessage}}
-        @resendStagedMessage={{this.resendStagedMessage}}
-      />
-    {{/each}}
+  <ChatMentionWarnings />
 
-    {{#if this.loadingMoreFuture}}
-      <ChatSkeleton @tagName="" />
-    {{/if}}
-  </div>
+  <div class="chat-message-actions-mobile-anchor"></div>
 
-  {{#if this.allPastMessagesLoaded}}
-    <div class="all-loaded-message">
-      {{i18n "chat.all_loaded"}}
-    </div>
-  {{/if}}
-</div>
+  <div
+    class={{concat-class
+      "chat-message-emoji-picker-anchor"
+      (if
+        (and
+          this.chatEmojiPickerManager.opened
+          (eq this.chatEmojiPickerManager.context "chat-message")
+        )
+        "-opened"
+      )
+    }}
+  ></div>
 
-{{#if this.showScrollToBottomBtn}}
-  <div class="scroll-stick-wrap">
-    <a
-      href
-      title={{i18n "chat.scroll_to_bottom"}}
-      class={{concat-class
-        "btn"
-        "btn-flat"
-        "chat-scroll-to-bottom"
-        (if this.hasNewMessages "unread-messages")
-      }}
-      {{on "click" (action "restickScrolling")}}
+  <div class="chat-messages-scroll chat-messages-container">
+    <div class="chat-message-actions-desktop-anchor"></div>
+    <div
+      class="chat-messages-container"
+      {{chat/track-highlighted this.didHighlightMessage}}
     >
-      {{#if this.hasNewMessages}}
-        {{i18n "chat.scroll_to_new_messages"}}
+      {{#if this.loadingMorePast}}
+        <ChatSkeleton />
+      {{/if}}
+
+      {{#each @channel.messages key="id" as |message|}}
+        <ChatMessage
+          @message={{message}}
+          @canInteractWithChat={{this.canInteractWithChat}}
+          @channel={{@channel}}
+          @setReplyTo={{this.setReplyTo}}
+          @replyMessageClicked={{this.replyMessageClicked}}
+          @editButtonClicked={{this.editButtonClicked}}
+          @selectingMessages={{this.selectingMessages}}
+          @onStartSelectingMessages={{this.onStartSelectingMessages}}
+          @onSelectMessage={{this.onSelectMessage}}
+          @bulkSelectMessages={{this.bulkSelectMessages}}
+          @isHovered={{eq message.id this.hoveredMessageId}}
+          @onHoverMessage={{this.onHoverMessage}}
+          @resendStagedMessage={{this.resendStagedMessage}}
```

Comment:
TODO: try this one

Reply:
Tried it locally using the Network tab to simulate no connection, worked fine.